attributes/default.rb

```diff
@@ -206,6 +206,7 @@
 default['cassandra']['opscenter']['server']['package_name'] = 'opscenter'
 default['cassandra']['opscenter']['server']['port'] = '8888'
 default['cassandra']['opscenter']['server']['interface'] = '0.0.0.0'
+default['cassandra']['opscenter']['server']['authentication'] = 'false' #default set to false
```

Comment:
this should also be `= false` instead of `= 'false'`, as the if condition will evaluate `'false'` as `true`